auth copy.py

```
@@ -0,0 +1,108 @@
+import hashlib
+import sqlite3
+from flask import Flask, request, jsonify
+import jwt
+import datetime
```

Ruff (0.12.2)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)